draft-poidt-ccamp-actn-poi-pluggable.md

```diff
@@ -237,29 +264,184 @@ Px = DWDM (coherent pluggable) Router ports
 # Reference architecture and network scenario
```

Comment:
This is clearly a reflection of solutions today and is not looking forward to where the solutions are likely to eventually land (single control fabric). That's OK so long as the intention of this document is to not push industry advancement.

It is expected that the single control fabric will be an assembly of fine-grained components from various vendors providing a holistic treatment of the entire network not fragmented by traditional artificial boundaries. It is also likely that there will be specialist components focused on key aspects such as photonic technology where these components have direct access to the relevant properties of the device (so a component dealing with photonic technology would access the device "directly" for monitoring and control of photonic parameters.

Reply:
v02

Reply:
OK. V02 is fine here. I would like to see more thinking wrt the target in V02.